requirements.txt

```diff
@@ -1,5 +1,12 @@
 asgiref==3.4.1
+cffi==1.14.6
+config==0.5.1
+cryptography==35.0.0
 Django==3.2.7
+django-appconf==1.0.5
+django-cryptography==1.0
+django-phone-field==1.8.1
+pycparser==2.20
 python-decouple==3.4
 pytz==2021.1
-sqlparse>=0.4.2
```

Comment:
replace with previous version "sqlparse>=0.4.2" to remove depreciation security error in GitHub